third_party/iap_jwt_verify_nginx/test/jwt_generation.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2002-2016 Igor Sysoev
+// Copyright (C) 2011-2016 Nginx, Inc.
+// Copyright (C) 2017 Google Inc.
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions
+// are met:
+// 1. Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+// 2. Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY THE AUTHOR AND CONTRIBUTORS ``AS IS'' AND
+// ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHOR OR CONTRIBUTORS BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+// OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+// HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+// LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+// SUCH DAMAGE.
+////////////////////////////////////////////////////////////////////////////////
+
+#ifndef NGINX_IAP_JWT_VERIFY_TEST_JWT_GENERATION_H
+#define NGINX_IAP_JWT_VERIFY_TEST_JWT_GENERATION_H
+
+#include <memory>
+
+#include "json/json.h"
```

Comment:
I'm not sure whether you're referring to the "json/json.h" include or this file as a whole, but...yes, both are needed. 

Assuming you meant the latter: though I don't actively generate JWTs in any of the tests,
1) it would be unfortunate to have to rewrite JWT generation code if we ever needed more
2) I'd like to keep open the option of building this test infrastructure out to support tests that leverage dynamic JWT generation (e.g. stress testing, or just fancier integration tests)